### PR TITLE
Use fuzzy tester module

### PR DESCRIPTION
**Pull request by orangejulius**
This removes the testing code from the acceptance-tests repo into the [pelias-fuzzy-tester](https://github.com/pelias/fuzzy-tester) repo. Also see https://github.com/pelias/fuzzy-tests/pull/4 for the same PR for the fuzzy-tests.

Would love feedback on the following things:
* I modeled the binary for the fuzzy tester after [ciao](https://github.com/missinglink/ciao). Any suggestions on overall layout are appreciated. This is my first npm module after all :)
  - in particular, the `fuzzy-tester` executable runs the test to run the tests now. Another option would be for the fuzzy-tester to behave like a library that is used in both repos.
* precommit-hook stuff, since I messed with it slightly
* README contents. Most of the instructions are now moved to the fuzzy-tester. Since it is all now technically part of the functionality of that package, it seems to make sense to keep it there and just point to it.
* Do we want a better name for the fuzzy-tester package? Clever names welcome

### CODE REVIEW

**Comment by dianashk** [comment]
Looks good so far. :+1:

File: README.md [hunk 1]
```diff
@@ -11,71 +11,10 @@ shotgun overview of the status of any Pelias instance.
 ## Usage
 
 ```
-node test --help
+npm test
```
Comment (dianashk) [inline README.md]: update with the `--` business?

**Comment by dianashk** [comment]
Other than the minor README update, it's good to go. :+1:

**Comment by orangejulius** [comment]
done!

On Wed, Jul 15, 2015 at 2:54 PM, Diana Shkolnikov <notifications@github.com>
wrote:

> Other than the minor README update, it's good to go. [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pelias/acceptance-tests/pull/114#issuecomment-121711352>
> .
>



-- 
Thanks,
Julian

**Pull request merged**